packages/rainbowkit/src/components/ConnectOptions/ConnectDetails.tsx

```diff
@@ -196,19 +200,21 @@ export function ConnectDetail({
     href?: string;
   } | null = showWalletConnectModal
     ? {
-        description: `Need the ${
-          compactModeEnabled ? '' : 'official'
-        } WalletConnect modal?`,
-        label: 'OPEN',
+        description: `${i18n.t('intro.wallet.get_started.need')} ${
+          compactModeEnabled ? '' : i18n.t('intro.wallet.get_started.official')
+        } ${i18n.t('intro.wallet.get_started.wallet_connect')}`,
```

Comment:
We need to bring these variables into the string for RTL languages/parsing; you can use multiple variables like this; just need to pass `modifier` to the i18n.t function
```
Need the %{modifier}WalletConnect modal?
```
Logic:
```ts
compactModeEnabled ? 'official ' : '' for proper spacing
```